python/cugraph/graph/c_graph.pyx

```diff
@@ -249,19 +249,58 @@ class Graph:
         
         offsets_data = rmm.device_array_from_ptr(offsets_col_data,
                                      nelem=off_size,
-                                     dtype=np.int32) # ,
+                                     dtype=np.int32)  # ,
                                      # finalizer=rmm._make_finalizer(offsets_col_data, 0))
         indices_data = rmm.device_array_from_ptr(indices_col_data,
                                      nelem=ind_size,
-                                     dtype=np.int32) # ,
+                                     dtype=np.int32)  # ,
                                      # finalizer=rmm._make_finalizer(indices_col_data, 0))
         # g.transposedAdjList.offsets.data and g.transposedAdjList.indices.data
         # are not owned by this instance, so should not be freed here (this
         # will lead to double free, and undefined behavior).
 
         return cudf.Series(offsets_data), cudf.Series(indices_data)
+
+    def get_two_hop_neighbors(self):
+        """
+        Return a dataframe containing vertex pairs such that each pair of vertices is 
+        connected by a path of two hops in the graph.
         
```

Comment:
We should mention if this is sorted or not.

Reply:
Done, it is sorted.

Reply:
Thanks for clarifying. Is it sorted by "src" and then by "dest" or is this an unstblabe sort? As far as I remember, jaccard implementation needed both to be sorted becasue of the binary search implementation.

If Jaccard needs pairs to be sorted, we should also say it in Jaccard doc. If not, we may need test on unsorted data.

Reply:
The inputs don't need to be sorted for Jaccard to work. Jaccard only depends on the adjacency of the graph being sorted, which it always is.